### DISCUSSION TITLE
show boundaries

### DISCUSSION THREAD

**Discussion by psknick** [Stellarium/stellarium#710] [Q&A] [answered]
Linux Mint 19.1 Cinnamon

In Stellarium-0.19.1-x86_64.AppImage  Problem: No boundaries of constellations:

![2019-07-06-14:12:45](https://user-images.githubusercontent.com/44753018/60755447-6fecb380-9ff8-11e9-9986-7d2954a08735.jpg)
![2019-07-06-14:11:58](https://user-images.githubusercontent.com/44753018/60755448-6fecb380-9ff8-11e9-8af4-e68a615b8643.jpg)

=======================
In Stellarium-0.18.3-x86_64.AppImage there are boundaries of constellations:

![2019-07-06-14:20:20](https://user-images.githubusercontent.com/44753018/60755493-3b2d2c00-9ff9-11e9-9536-c40888b75c67.jpg)

**Comment by alex-w** [comment]
I've tested 0.19.1 AppImage right now and I do not see any problem for boundaries. Please attach log from your copy of planetarium.

**Comment by psknick** [comment]
[log.txt](https://github.com/Stellarium/stellarium/files/3364654/log.txt)

**Comment by alex-w** [comment, marked as answer]
Please run in terminal: sudo apt purge stellarium -y

You are mixed 2 version of Stellarium on your system. By default AppImage checked standard path for application and if it exists then AppImage use it. Seems you have installed version 0.18 from repository already.
